esmvaltool/diag_scripts/hydrology/marrmot.py

```diff
@@ -79,6 +79,20 @@ def _get_extra_info(cube):
     return time_start_end, lat_lon
 
 
+def fix_era5_time_coordinate(cube):
```

Comment:
```suggestion
def fix_era5_time_coordinate(cube, shift=30):
```
Not strictly necessary, but perhaps it could be nice for future use cases?

Reply:
@Peter9192 thanks. I didn't to apply this suggestion because I think the shift value (here 30min) is not something that changes in the future.